File: LeetCode/1491-LeetCode.js
// 1491. Average Salary Excluding the Minimum and Maximum Salary
// Difficulty: Easy

/**
 * @param {number} low
 * @param {number} high
 * @return {number}
 */
var countOdds = function(low, high) {
    let num = Math.floor((high - low)/2);
    if (low%2 || high%2) {
        num++;
    }
    
    return num;
};

File: LeetCode/1491-LeetCode.py
# 1491. Average Salary Excluding the Minimum and Maximum Salary
# Difficulty: Easy

class Solution:
    def average(self, salary: List[int]) -> float:
        if len(salary)<=2:
            return float(0)
        min = salary[0]
        max = salary[0]
        sum = 0
        
        for i in range(0,len(salary)):
            if salary[i]>max:
                max = salary[i]
            if salary[i]<min:
                min = salary[i]
            
            sum = sum + salary[i]
            
        return float((sum-max-min)/(len(salary)-2))
